docs/examples.rst

```diff
@@ -665,4 +665,15 @@ Finally, you can exclude single attributes from certain methods:
    ...     user = attr.ib()
    ...     password = attr.ib(repr=False)
    >>> C("me", "s3kr3t")
-   C(user='me')
+   C(user='me', password=***)
+
+Alternatively, to influence how the generated ``__repr__()`` method formats a specific attribute, specify custom callable to be used instead of the `repr()` built-in function:
```

Comment:
specify "a"

Reply:
fixed in f6b766f1ada5f8791738b51d851076695d64645c